### PR TITLE
Updates to cluster profiles

### PR DESCRIPTION
**Pull request by ritawatson**
## Describe the Change

This PR addresses comments in this [slack 🧵](https://spectrocloud.slack.com/archives/C04RY1ZEWN9/p1698080632328189). 

The PR fixes a caution rendering, revises the introductory sentences in cluster profile add-on docs (packs, manifests, and Helm charts), adds a caution to recommend versioning a profile versus updating it, and retitles a doc to remove the term "Duplicate".

## Review Changes

💻 Updated pages:

- [Profile Layers](https://deploy-preview-1721--docs-spectrocloud.netlify.app/profiles/cluster-profiles/#profile-layers) - Added sentence to the first paragraph about adding a pack multiple times to the same profile.
- [Add a Pack](https://deploy-preview-1721--docs-spectrocloud.netlify.app/profiles/cluster-profiles/create-cluster-profiles/create-addon-profile/create-pack-addon) - Revised first sentence.
- [Add a Manifest](https://deploy-preview-1721--docs-spectrocloud.netlify.app/profiles/cluster-profiles/create-cluster-profiles/create-addon-profile/create-manifest-addon) - Revised first sentence.
- [Add a Helm Chart](https://deploy-preview-1721--docs-spectrocloud.netlify.app/profiles/cluster-profiles/create-cluster-profiles/create-addon-profile/create-helm-addon) - Revised first sentence.
- [Deploy Same Pack to Multiple Layers](https://deploy-preview-1721--docs-spectrocloud.netlify.app/profiles/cluster-profiles/create-cluster-profiles/duplicate-pack-in-profile) - Retitle
- [Modify Cluster Profiles](https://deploy-preview-1721--docs-spectrocloud.netlify.app/profiles/cluster-profiles/modify-cluster-profiles/) - Added a caution.
- [Update a Cluster Profile](https://deploy-preview-1721--docs-spectrocloud.netlify.app/profiles/cluster-profiles/modify-cluster-profiles/update-cluster-profile) - Added a caution.

### CODE REVIEW

File: docs/docs-content/profiles/cluster-profiles/modify-cluster-profiles/update-cluster-profile.md [hunk 1]
```diff
@@ -12,6 +12,12 @@ You can update a cluster profile to change the configuration of one or more laye
 
 Cluster profile changes will generate an update notification on all the clusters that are created from the profile. Update notifications include information about all the changes applied to the profile since the initial creation or since the previous update. You can apply cluster updates individually at any time.
 
+:::caution
+
+We do not recommend updating cluster profiles that have been used to deploy clusters. Instead, we recommend creating a new profile version and pushing out the changes to the running clusters. For information on versioning profiles, check out [Version a Cluster Profile](version-cluster-profile.md).
```
Comment (antongisli) [inline docs/docs-content/profiles/cluster-profiles/modify-cluster-profiles/update-cluster-profile.md:17]: @ritawatson @karl-cardenas-coding this one is a little tricky to get right. What we don't recommend is updating a currently deployed cluster profile _version_ in order to push out new changes. Instead, we recommend creating a new version ...  I'll let you decide how to word smith that.

**Review by antongisli — CHANGES_REQUESTED** [review]
apart from one comment that needs attention, LGTM

File: docs/docs-content/profiles/cluster-profiles/cluster-profiles.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ The diagram below illustrates the components of these profile types and how you
 
 ## Profile Layers
 
-Each cluster profile layer provides a specific functionality. Profile layers can be packs, Helm Charts, or custom manifests, as described in the table.
+Each cluster profile layer provides a specific functionality. Profile layers can be packs, Helm Charts, or custom manifests, as described in the table. It is possible to add a pack multiple times to the same cluster profile. To learn more about this scenario, review [Deploy Same Pack to Multiple Layers](../cluster-profiles/create-cluster-profiles/duplicate-pack-in-profile.md).
```
Comment (karl-cardenas-coding) [inline docs/docs-content/profiles/cluster-profiles/cluster-profiles.md:26]:
```suggestion
Each cluster profile layer provides a specific functionality. Profile layers can be packs, Helm Charts, or custom manifests, as described in the table. Adding a pack multiple times to the same cluster profile is possible. To learn more about this scenario, review [Deploy Same Pack to Multiple Layers](../cluster-profiles/create-cluster-profiles/duplicate-pack-in-profile.md).
```
More of an active voice.

File: docs/docs-content/profiles/cluster-profiles/modify-cluster-profiles/modify-cluster-profiles.md [hunk 3]
```diff
@@ -14,9 +14,14 @@ You can modify cluster profiles by updating the following:
 
 - Profile layers or pack versions.
 
-You can also create multiple versions of a cluster profile with different pack configurations. For more information, review the [Version a Cluster Profile](version-cluster-profile.md) guide.
+  :::caution
 
-In the event that more than one application in the profile needs the same pack, Palette allows you to deploy the same pack to multiple layers. Check out [Duplicate a Pack in a Profile](../create-cluster-profiles/duplicate-pack-in-profile.md) for more information.
+  If you want to update cluster profiles that have been used to deploy clusters, we recommend creating a new profile version and pushing out the changes to the running clusters. To learn how to version profiles, review [Version a Cluster Profile](version-cluster-profile.md) guide.
```
Comment (karl-cardenas-coding) [inline docs/docs-content/profiles/cluster-profiles/modify-cluster-profiles/modify-cluster-profiles.md:19]:
```suggestion
  If you want to update cluster profiles that have been used to deploy clusters, we recommend creating a new profile version and pushing out the changes to the active clusters. To learn how to version profiles, review [Version a Cluster Profile](version-cluster-profile.md) guide.
```

**Review by karl-cardenas-coding — APPROVED** [review]
Looks good @ritawatson - just one suggestion and I'll defer to you to wordsmith Anton's comment about deployed cluster profiles.

File: docs/docs-content/profiles/cluster-profiles/modify-cluster-profiles/update-cluster-profile.md (hunk 1 above)
Comment (ritawatson) [reply docs/docs-content/profiles/cluster-profiles/modify-cluster-profiles/update-cluster-profile.md:17]: @antongisli I see the nuance. Revised.

Comment (ritawatson) [reply docs/docs-content/profiles/cluster-profiles/modify-cluster-profiles/update-cluster-profile.md]: Revised as follows: 

CleanShot 2023-11-10 at 12 09 54

**Pull request merged**